common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/protocol/FetchShuffleBlocks.java

```diff
@@ -36,19 +36,24 @@
   // it corresponds to the i-th int[] in reduceIds, which contains all reduce id for this map id.
   public final long[] mapIds;
   public final int[][] reduceIds;
+  // Indicates whether read continuous shuffle blocks in batch for IO reducing. When it is true,
+  // the reduceIds contains the range [startReduceId, endReduceId) for each ShuffleBlockBatchId.
```

Comment:
I think it's clearer to document `reduceIds` instead of `batchFetchEnabled`. For example,
```
when batchFetchEnabled=true, reduceIds[i] contains 2 elements: startReduceId (inclusive) and
endReduceId (exclusive) for the mapper mapIds[i].
when batchFetchEnabled=false, reduceIds[i] contains all the reduce IDs that mapper mapIds[i] needs to fetch.
```

Reply:
Thanks for the new comment, done in 407b1e0.